### DISCUSSION TITLE
Unlink seed from prompt broken?

### DISCUSSION THREAD

**Discussion by Weatherby43** [adieyal/sd-dynamic-prompts#681] [Q&A]
Hi,
I was wondering: Of late, when I lock a seed for a prompt that has wildcards in it, I no longer get the same image. I have tried reinstalling the extension, I have tried deleting the venv directory and reinstalling. The problem seems to be that each time an image is generated it re-evaluates the wildcards instead of locking them the way it did before? I do not know if this happened wiht an update to the extension or not, but I tried out the LCM image generation procedures, and that's when I noticed that a locked seed no longer generated the same image. I frequently use this on my 3060 due to the time it takes to generate a highres fix pass. I generate images at the base resolution, then if I find one I like, I lock the seed and enable highres fix and regenerate. Up until about a week ago this worked just fine, but now locking the seed does nothing unless I copy the full parsed prompt. If I use the one with the wildcards evaluated and inserted into the prompt, I get the same image, but that's an exatra step and I have to replace the prompt over and over when I did not have to do this before. Reading the description of the unlink seed from prompt in advanced settings it seems to be functioning as if that's enabled.. by generating random prompts even with a locked seed...  Any help would be appreciated! Thanks!

EDIT:

Update: I just tried {x|Y|Z} style randomization and locked the seed. The image was the same. So it's only __wildcard__ type usage that seems to be not functioning correctly.

**Comment by ProjectionistFM** [comment]
I use the exact same workflow and am now having the same issue. Previously, I could generate with wildcards in the prompt. When I found one I liked, I would lock the seed and highres fix it. Now when I do that, the wildcards are updated, even though the seed is fixed and I get a different image.
